sdk/servicebus/azure-messaging-servicebus/pom.xml

```diff
@@ -44,6 +44,11 @@
       <artifactId>azure-core</artifactId>
       <version>1.10.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-experimental</artifactId>
+      <version>1.0.0-beta.9</version> <!-- {x-version-update;com.azure:azure-core-experimental;current} -->
```

Comment:
This should use `unreleased_` tag since beta.9 is not yet released. Also, the last component of the version tag should be `dependency` not `current`.

Reply:
done in separate PR